src/storage/btree.c

```diff
@@ -16424,8 +16493,6 @@ btree_find_min_or_max_key (THREAD_ENTRY * thread_p, BTID * btid, DB_VALUE * key,
 
   pgbuf_unfix_and_init (thread_p, root_page_ptr);
 
-  assert (tp_valid_indextype (TP_DOMAIN_TYPE (BTS->btid_int.key_type)));
```

Comment:
Why did you delete it?

Reply:
This check if the index is sing-key type, we should permmit multi-key index. So, I remove the check routine.